app/controllers/application_controller.rb

```diff
@@ -56,6 +56,7 @@ def verify_private_forem
     if api_action?
       authenticate!
     else
+      @user ||= User.new
```

Comment:
this solves the new/create/new flow I think (@user is now never nil when rendering the partial)